.github/workflows/multicluster.yaml

```diff
@@ -0,0 +1,247 @@
+name: Multicluster / Cluster mesh (ci-multicluster)
+
+on:
+  issue_comment:
+    types:
+      - created
+  push:
+    branches:
+      - master
+  ### FOR TESTING PURPOSES
+  # pull_request:
+  #  types:
+  #    - "labeled"
+  ###
```

Comment:
Rather, it *is* for debugging/developing (we've intentionally left these in in all `issue_comment` workflows so that workflow developers can uncomment them when needed, and also making explicit that the `pull_request` triggers should NOT be merged uncommented).